CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+## [1.3.1] - 2024-02-13
+
+- Change `TARANTOOL_VERSION` default value from `2.11.2-centos7` to `2.11.2-ubuntu20.04` and update Dockerfile
```

Comment:
We need to add the reason of these changes here and to commit and to PR description. About why we do it

Reply:
OK, I'll do it